packages/react-cards/package.json

```diff
@@ -48,10 +49,20 @@
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
+    "@fluentui/keyboard-key": "^0.2.12",
+    "@fluentui/react-compose": "^0.19.3",
+    "@fluentui/react-theme-provider": "^0.13.3",
+    "@fluentui/theme": "^0.3.2",
     "@microsoft/load-themed-styles": "^1.10.26",
+    "@uifabric/azure-themes": "^7.5.13",
```

Comment:
wait, this should not be a dependency.

Reply:
check the others as well.

Reply:
Spoke about this offline with David. A lot of this cleanup is already being done in #15132.